zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteringRule.java

```diff
@@ -215,6 +219,12 @@ public ClusteringRule(
             .collect(Collectors.toList());
 
     clusterName = "zeebe-cluster-" + CLUSTER_COUNT.getAndIncrement();
+    meterRegistry = new SimpleMeterRegistry();
+    closeables.add(
+        () -> {
+          meterRegistry.clear();
```

Comment:
:wrench:  use `MicrometerUtil.closeRegistry`

Reply:
I removed it as part of using the partition startup registry, as long as it's not used in production code, I would still keep that out.